### ISSUE TITLE
Running app fails with ERR_OSSL_EVP_UNSUPPORTED

### ISSUE THREAD

**Issue by olifante** [jsdisco/react-guitar#1]
Running `npm run start` or `yarn start` fails with the following error:

```
Error: error:0308010C:digital envelope routines::unsupported
[...]
    at [...]/node_modules/babel-loader/lib/index.js:59:103 {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
```

OS: macOS Monterey 12.5.1 running on M1 processor
node: v17.0.1
npm: 8.19.3

**Comment by marayana** [comment]
> Running `npm run start` or `yarn start` fails with the following error:
> 
> ```
> Error: error:0308010C:digital envelope routines::unsupported
> [...]
>     at /Users/tiago/Documents/GitHub/GUITAR/react-guitar-hubot/node_modules/babel-loader/lib/index.js:59:103 {
>   opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
>   library: 'digital envelope routines',
>   reason: 'unsupported',
>   code: 'ERR_OSSL_EVP_UNSUPPORTED'
> }
> ```
> 
> OS: macOS Monterey 12.5.1 running on M1 processor node: v17.0.1 npm: 8.19.3

You're correct, I can reproduce it. Given that the repo is two years old, I'm sure *something* is deprecated. I'll look into it as soon as possible, thanks for the heads up.

**Comment by marayana** [comment]
Running`npm update` fixed it for me (MacOS Ventura 13.0.1, Node 17.7.2, npm 8.5.2). 

Some further information/possible fixes that might help: [https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported](url) (I'd rather not downgrade my Node version though).
